book/website/ethical-research/activism/activism-env-impact.md

```diff
@@ -0,0 +1,165 @@
+
+The Climate Aware Task Scheduler (CATS)[[https://github.com/GreenScheduler/cats](https://github.com/GreenScheduler/cats)]) has been built specifically with this in mind. This tool can calculate how much carbon will be emitted during the run of a specific task, look at the carbon emission forecast, and schedule the task to be run at a time when carbon intensity is low.
+
+```{figure} ../../figures/carbon_computing.*
+---
+height: 500px
+name: Computing has carbon emissions
+alt: Computing has carbon emissions
+---
+Computing has carbon emissions
+```
+
+(er-activism-env-impact-data)=
+### Data
+The increasing number of digital research and the associated storage requirements have implications for the environment, and understanding the environmental impact is key for sustainable scientific research practices. One important aspect to consider is the environmental impact of digital storage:
+
+(er-activism-env-impact-data-reduction)=
+#### Data reduction 
+Reduction of data volumes is a straight-forward solution to minimize energy consumption in storage systems:
+* Compression or deduplication can help to detect and delete repeated information. 
+* Optimized data requires less time for transfer and consumes less network bandwidth.
+
+(er-activism-env-impact-data-standardisation)=
+#### Standardisation
+Storing your data in standardised data format can have a positive impact on the environment:
+* Utilize data formats that are widely accepted within your community to prevent the need for conversion by those interested in reusing your data.
+* Similarly, embrace standardized variable names and, if applicable to your data, employ standardized physical units.
+
+In addition, deposit your datasets in domain specific or community archives so that users ware promptly informed about the availability of your dataset. This can also avoid duplication of effort and recomputation/acquisition of the same data.
+
+(er-activism-env-impact-data-green-centers)=
+#### Green Data Centers
+The usage of shared computing and storage infrastructure is usually a way to reduce the impact of data storage in the environment. The reason is that most data centres invest in energy-efficient servers, storage systems, and networking equipment. This includes using hardware components with high energy efficiency ratings and employing advanced cooling techniques to reduce power consumption. In addition, the implementation of virtualization technologies allows for better usage of server resources.
+
+Finally some data centers are shifting towards renewable energy sources (solar, wind, hydroelectric power) and reduce their dependencies on fossil fuels.
+
+The greenest data/HPC centers are listed on the [GREEN500 list](https://www.top500.org/lists/green500/).
+
+Whenever you can choose, select data centers that are committed to reduce their carbon footprint.
+
+### Other resources
+#### Computation

```

Comment:
I suggest to add short titles to each of bullet point (see for example other resources for data).

Reply:
Very good point -- adding a suggestion below, does that look ok?

Reply:
Yes, it looks great!